### ISSUE TITLE
bash_completion: nosort: invalid option name

### ISSUE THREAD

**Issue by ilovezfs** [ocaml/opam#2851]
Testing out 2.0.0-beta2, I noticed that there was a problem with the new bash completions when using them on macOS with the system bash 3.x (specifically on 10.11 /bin/bash is GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin15)) since it doesn't understand the new `nosort` option, leading to

```
Josephs-MacBook-Pro:~ joe$ ssh 10.0.1.15
Last login: Tue Feb 14 05:04:30 2017 from 10.0.1.83
-bash: complete: nosort: invalid option name
```

**Comment by AltGr** [comment]
Ah, too bad, thanks for reporting!
I'll remove it since it doesn't seem to work as I expected anyway.
(I was hoping to get main commands before flags, and flags before files, but it's not what got out)

**Comment by AltGr** [comment]
Fixed by #2852
